test/Concurrency/Runtime/data_race_detection.swift

```diff
@@ -15,6 +15,9 @@
 import _Concurrency
 import Dispatch
 
+// FIXME(Concurrency): We cannot make the _taskIsCurrentExecutor crashing; must move to `_taskIsOnExecutor`
+// XFAIL: *
```

Comment:
FIXME: until we make the `swift_task_isOnExecutorImpl` the API we call from Swift APIs, this is going to be blocked;

We cannot use the `_taskIsCurrentExecutor` (`swift_task_isCurrentExecutor`) as we do today to make it the "crashing one" because this will also crash in this "data race detector" which expects to be able to log warnings. But our APIs cannot support this.